tests/broken_specimens.yaml

```diff
@@ -0,0 +1,70 @@
+study:
```

Comment:
Feels like the pre-commit hook should have been a separate PR to this - but i'm not against including bad yamls for future testing.

Reply:
Yeah, bad yaml is intentionally here,